open-metadata-implementation/user-interfaces/ui-admin-services/ui-admin-services-api/src/main/java/org/odpi/openmetadata/userinterface/adminservices/rest/PlatformSecurityRequestBody.java

```diff
@@ -0,0 +1,99 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.userinterface.adminservices.rest;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.odpi.openmetadata.frameworks.connectors.properties.beans.Connection;
+
+import java.util.Objects;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.PUBLIC_ONLY;
+
+/**
+ * PlatformSecurityRequestBody passes information to set up a security connector to protect
```

Comment:
There is no reason why this needs to be different from the server platform

Reply:
Removed this file